### DISCUSSION TITLE
[Feature] Nacos supports configuration context path

### DISCUSSION THREAD

**Discussion by wetool19** [apache/skywalking#12557] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Nacos supports configuration context path
![image](https://github.com/user-attachments/assets/55390df1-5188-4fdb-ad28-b50961caed42)

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I don't know what do you mean. THere is no context path thing.

**Comment by wetool19** [comment]


**Reply by wu-sheng** [reply]
Please keep in English on the GitHub. I have to remove your previous comments.

And if you want to add this, I think it is easy to submit a pull request directly.

**Comment by wu-sheng** [comment]
Followed here, https://github.com/apache/skywalking/pull/12558
